ee/rpc/audsrv/include/audsrv.h

```diff
@@ -221,9 +221,10 @@ int audsrv_adpcm_init();
 /** Sets output volume for the specified voice channel.
  * @param ch       Voice channel ID
  * @param vol      volume in percentage (0-100)
+ * @param pan      left/right offset [-100 .. 0 .. 100]
  * @returns 0 on success, negative otherwise
  */
-int audsrv_adpcm_set_volume(int ch, int vol);
+int audsrv_adpcm_set_volume(int ch, int vol, int pan);
```

Comment:
in order to keep retro compatibility, should we create another method instead? and just call that method with the default value for the existing function?

Reply:
was thinking the same.. https://github.com/ps2dev/ps2sdk/blob/master/ee/rpc/audsrv/include/audsrv.h#L247
for example

Reply:
Done

Reply:
Welll done!